appshell/cefclient_mac.mm

```diff
@@ -334,8 +334,15 @@ - (void)windowWillEnterFullScreen:(NSNotification *)notification {
     }
     if (customTitlebar) {
         [customTitlebar setHidden:YES];
+        NSWindow *window = [notification object];
+
+        // Since we have nuked the title, we will have
+        // to set the string back as we are hiding the
+        // custom title bar.
+        [window setTitle:[customTitlebar titleString]];
```

Comment:
There seems to be an issue with custom title bar view in case "Debug > New Brackets Window" is selected and the same is maximized and/or restored. See screenshot below -
![screen shot 2016-05-12 at 11 04 14 pm](https://cloud.githubusercontent.com/assets/2939847/15224257/f9a34906-1895-11e6-92fd-b7e70e99892a.png)